<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavestatesDialog</class>
 <widget class="QDialog" name="SavestatesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unsupported game savestates</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uiFilterLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="uiSavestatesTblv"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="uiButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uiFilterLineEdit</tabstop>
  <tabstop>uiSavestatesTblv</tabstop>
  <tabstop>uiButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>uiButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SavestatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SavestatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
